overlord/snapstate/snapmgr.go

```diff
@@ -390,39 +390,37 @@ func refreshScheduleNoWeekdays(rs []*timeutil.Schedule) error {
 	return nil
 }
 
-func (m *SnapManager) checkRefreshSchedule() ([]*timeutil.Schedule, error) {
+func resetRefreshScheduleToDefault(st *state.State) (ts []*timeutil.Schedule, scheduleStr string) {
+	refreshSchedule, err := timeutil.ParseSchedule(defaultRefreshSchedule)
```

Comment:
Would it make sense to add `timeutil.MustParseSchedule()` similarly to what we have in `regexp`?

Reply:
Yes, I think that is sensible - I'm just too lazy right now as its the only place where we need this :)